assistant/Dockerfile

```diff
@@ -1,8 +1,7 @@
-FROM python:slim
+FROM python:3.12.3
 
 RUN pip install pika ollama
```

Comment:
Should use `requirements.txt` to benefit from version locking in that file.

Reply:
done